listenbrainz/webserver/templates/api_compat/index.html

```diff
@@ -0,0 +1,5 @@
+<html>
+  <head> <title> LB API Compat Index Page </title> </head>
+
+  <body> Please visit https://listenbrainz.org </body>
```

Comment:
We should have a page with documentation on how to use this (and the other) API. Let's create such a page and then redirect from here to that page.

Reply:
Sure, makes sense. I kept this minimal because https://post.audioscrobbler.com/ was pretty small, I'lll change it now.